Packs/Core/Playbooks/playbook-Suspicious_LDAP_search_query.yml

```diff
@@ -0,0 +1,825 @@
+    scriptarguments:
+      process_command_line:
+        simple: ${alert.osparentcmd}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1150,
+          "y": 270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 9fcad966-4a26-431d-8436-8b454fe7d3f1
+    type: regular
+    task:
+      id: 9fcad966-4a26-431d-8436-8b454fe7d3f1
+      version: -1
+      name: Get Host Score from Risky Assets
+      description: Retrieve the risk score of a specific host or list of hosts with the highest risk score in the environment along with the reason affecting each score.
+      script: '|||core-list-risky-hosts'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      host_id:
+        simple: ${alert.hostname}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1580,
+          "y": 270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 7d77d223-491a-4df0-8da6-03dd1a68408a
+    type: title
+    task:
+      id: 7d77d223-491a-4df0-8da6-03dd1a68408a
+      version: -1
+      name: Without the Risky Score
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 240,
+          "y": 635
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: 18445382-e12f-4e92-81f9-d615f319048d
+    type: title
+    task:
+      id: 18445382-e12f-4e92-81f9-d615f319048d
+      version: -1
+      name: With Risky Score
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 750,
+          "y": 715
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: c1f72153-94cd-4322-8f42-2c33e6c320e6
+    type: regular
+    task:
+      id: c1f72153-94cd-4322-8f42-2c33e6c320e6
+      version: -1
+      name: Terminate the Causality Process Manually
+      description: |
+        Dear Analyst,
```

Comment:
Please update the description to:

Dear Analyst,

During the remediation process, the playbook couldn’t terminate the process: ${alert.cgoname}

Please terminate the process manually if possible.